io.github.tareksander.story_reader.yaml

```
@@ -0,0 +1,30 @@
+id: io.github.tareksander.story_reader
+runtime: org.freedesktop.Platform
+runtime-version: "23.08"
+command: story_reader
+sdk: org.freedesktop.Sdk
+finish-args:
+- "--share=network"
+- "--socket=pulseaudio"
+- "--socket=fallback-x11"
+- "--socket=wayland"
+- "--share=ipc"
+- "--device=dri"
+
+modules:
+  - name: "x86_64_binary_bundle"
```

```suggestion
  - name: "story_reader"
```

Really naming things isn't hard.